### PR TITLE
task9 

### PR DESCRIPTION
**Pull request by PriyankaChaganti**

### CODE REVIEW

File: robust_svm/main.py [hunk 1]
```diff
@@ -9,11 +9,20 @@
 #print(platform.uname())
```
Comment (sahithi403) [inline robust_svm/main.py]: remove this old code.  create a new file and add it to temp_space folder if you like to save the code for future use

Comment (PriyankaChaganti) [reply robust_svm/main.py]: under file_priyanka??

File: .gitignore [hunk 2]
```diff
@@ -105,3 +105,8 @@ venv.bak/
 
 # Pycharm
 .idea*
+*.ppm
+*.jpg
+*.ppm
+*.ppm
```
Comment (sahithi403) [inline .gitignore]: why do you have multiple entries for .ppm files? 

To ignore all files in a folder 'data', you need to add the folder path to .gitignore as follows 
/data/*

Comment (PriyankaChaganti) [reply .gitignore]: it worked!

Comment (PriyankaChaganti) [reply .gitignore]: multiple trials

File: robust_svm/main.py (hunk 1 above)
Comment (sahithi403) [reply robust_svm/main.py]: filename doesn't matter. its for your personal use

File: robust_svm/main.py [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
-#print(cv2.__version__)
-path="C:/Users/ch.srivamsi priyanka/Documents/GitHub/traffic_sign_svm/data/training_data/*.*"
+path="C:/Users/ch.srivamsi priyanka/Documents/GitHub/traffic_sign_svm/data/training_data/Images/*/*.*"
 for file in glob.glob(path):
  a = cv2.imread(file)
  c = cv2.cvtColor(a, cv2.COLOR_BGR2RGB)
  cv2.imshow('Color image', c)
- cv2.waitKey(10000)
+ cv2.waitKey(100)
  cv2.destroyAllWindows()
```
Comment (sahithi403) [inline robust_svm/main.py]: you don't need lines 1-10 right now. move it into your temporary file

Comment (PriyankaChaganti) [reply robust_svm/main.py]: made changes

File: robust_svm/main.py [hunk 4]
```diff
@@ -1,19 +1,10 @@
...
-#print(datetime.datetime.now())
-#print(cv2.__version__)
-path="C:/Users/ch.srivamsi priyanka/Documents/GitHub/traffic_sign_svm/data/training_data/*.*"
-for file in glob.glob(path):
- a = cv2.imread(file)
- c = cv2.cvtColor(a, cv2.COLOR_BGR2RGB)
- cv2.imshow('Color image', c)
- cv2.waitKey(10000)
- cv2.destroyAllWindows()
+training_data=make_dataset(folder_path,folders_list)
```
Comment (sahithi403) [inline robust_svm/main.py:2]: you need to import 'make_datasset' from read_images.py. Accepting this PR for now. Do it in your next PR

**Review by sahithi403 — APPROVED** [review]
you need to import 'make_datasset' from read_images.py. Accepting this PR for now. Do it in your next PR

**Pull request merged**